plugins/events-backend-module-sqs/README.md

```diff
@@ -0,0 +1,25 @@
+# events-backend-module-sqs
+
+Welcome to the events-backend-module-sqs backend plugin!
+
+This plugin is a module for the event-backend backend plugin
+and extends it with an `AwsSqsConsumingEventPublisher`.
+
+This event publisher will allow you to receive events from
+an AWS SQS queue and will publish these to the used event bus.
+
+The polled AWS SQS queues depend on your configuration:
+
+```yaml
+events:
+  modules:
+    awsSqs:
+      topics:
```

Comment:
I would not call it consumeTopics as we don't consume the topics / subscribe to them but publish to them.

`awsSqsConsumer` seems reasonable though.

Reply:
hmm.. `events.modules.{moduleName}` would rather point to use `sqs` (or `awsSqs` when adjusting the module name which I've considered anyways).

maybe `events.modules.awsSqs.awsSqsConsumingPublisher` so that it is easier to extend later.

Reply:
I went for `events.modules.awsSqs.awsSqsConsumingEventPublisher` -- maybe a bit long though.